ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,11 @@
+
+#### This issue is about [Algo/DS Name](link to resource for the Algo/DS).
+
+- [ ] Issue Title is in form `Algo/DS Name [Language]`.
+- [ ] I searched or browsed the repo’s other issues to ensure this is not a duplicate.
+- [ ] This Algo/DS is worth adding to this repository.
+- [ ] Assign this issue to me (I want to work on this).
+- [ ] I will send my PR only after I'm assigned to this issue by any maintainer.
+
+##### It's not necessary for all for points to be checked, we will look upon what is required, once you open an issue.
```

Comment:
Also this line can be a comment. (See PR template if you are not sure what comment is.)